microsetta_public_api/models/_taxonomy.py

```diff
@@ -130,8 +130,7 @@ def __init__(self, table: biom.Table, features: pd.DataFrame,
                 set(self._table.ids(axis='observation')):
             raise DisjointError("Table and variances are disjoint")
 
-        if set(self._table.ids(axis='observation')) != \
-                set(self._features.index):
+        if not self._feature_id_lookup.issubset(set(self._features.index)):
             raise DisjointError("Table and features are disjoint")
```

Comment:
Maybe we should change this error/message? Disjoint would indicate that there is not overlap between the table and the feature data.

Reply:
Good call, thanks! I'll address shortly